base/shell/explorer/startmnucust.cpp

```diff
@@ -66,16 +66,75 @@ VOID OnClearRecentItems()
         }
         while (FindNextFileW(hPath, &info));
         FindClose(hPath);
-        /* FIXME: Disable the button*/
+        
+        EnableWindow(GetDlgItem(hwnd, IDC_CLASSICSTART_CLEAR), FALSE);
     }
 }
 
+INT_PTR CALLBACK CustomizeModernProc(HWND hwnd, UINT Message, WPARAM wParam, LPARAM lParam)
+{
+    switch(Message)
+    {
+        case WM_INITDIALOG:
+        return TRUE;
+        case WM_COMMAND:
+            switch(LOWORD(wParam))
+            {
+                case IDOK:
+                    EndDialog(hwnd, IDOK);
+                break;
+                case IDCANCEL:
+                    EndDialog(hwnd, IDCANCEL);
+                break;
+            }
+        break;
+        default:
+        return FALSE;
+    }
+    return TRUE;
+}
+
+HWND InitClassicListView(HWND hwnd)
+{
+    LVCOLUMN lvc = {0};
+    RECT rcClient;
+
+    HWND hListView = GetDlgItem(hwnd, IDC_CLASSICSTART_SETTINGS);
+    ListView_SetExtendedListViewStyle(hListView, LVS_EX_CHECKBOXES | LVS_EX_FULLROWSELECT);
+
+    GetClientRect(hListView, &rcClient);
+    lvc.mask = LVCF_TEXT | LVCF_WIDTH;
+    lvc.pszText = (LPWSTR)_T("");
```

Comment:
Unnecessary cast here. Just explicitly initialize your `pszText` member with `L""` instead (do not forget the `lvc` variable has to be declared using the -W Unicode convention, `LVCOLUMNW`, at that point).

Reply:
The cast is needed, but the text should be `L""`